docs/src/main/paradox/bootstrap.md

```diff
@@ -245,15 +231,7 @@ file for the service configuration, which is:
 
 @@snip [akka-cluster.yml](../../../../bootstrap-joining-demo/kubernetes-api/kubernetes/akka-cluster.yml) 
 
-In order to run the example you can build it via:
-
-```
-sbt shell
-> project bootstrap-joining-demo
-> docker:publishLocal
-```
```

Comment:
This fails if minikube has not been started first, and the same thing is anyway repeated below...

Reply:
good catch ,thanks